scout/server/blueprints/cases/views.py

```diff
@@ -433,6 +434,19 @@ def gene_variants(institute_id):
     return dict(institute=institute_obj, form=form, page=page, **data)
 
 
+@cases_bp.route('/<institute_id>/<case_name>/individuals', methods=['POST'])
+def update_individual(institute_id, case_name):
+    """Update individual data (age and/or Tissue type) for a case"""
+
+    institute_obj, case_obj = institute_and_case(store, institute_id, case_name)
+    user_obj = store.user(current_user.email)
+    ind_id = request.form.get('update_ind')
+    age = request.form.get('_'.join(['age',ind_id]))
+    tissue = request.form.get('_'.join(['tissue',ind_id]))
```

Comment:
Not saying it must be so, but wouldn't it be kind of par with an event for that? On one hand it is just a note, but it would be used for sample aggregation at least, so might be good to have some audit on it?

Reply:
It is a very good idea, I'll introduce it!